1_fetch.R

```diff
@@ -88,8 +88,35 @@ p1_targets_list <- list(
     p1_inst_timeseries_summary_csv,
     command = target_summary_stats(p1_inst_data,"Value_Inst","1_fetch/log/inst_timeseries_summary.csv"),
     format = "file"
-  )
+  ),
   
-)
-
+  tar_target(
+    p1_reaches_shp_zip,
+    # [Jeff] I downloaded this manually from science base: 
+    # https://www.sciencebase.gov/catalog/item/5f6a285d82ce38aaa244912e
+    # Because it's a shapefile, it's not easily downloaded using sbtools
+    # like other files are (see https://github.com/USGS-R/sbtools/issues/277).
```

Comment:
Thanks for pointing to this sbtools issue - it confirms an issue I've run into before. I wonder if there are any plans to add functionality for pulling down shp files

Reply:
IDK if there are plan for adding that functionality, but I think Blodgett made the issue (https://github.com/USGS-R/sbtools/issues/277) and tagged it `enhancement` so that hopefully one day they will make it easier to do these downloads.